cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -133,43 +133,23 @@ void generate_short_reads(std::vector<std::vector<std::string>>& windows, BatchS
     batch_size = BatchSize(1024, 100);
 }
 
-void generate_simulated_long_reads(std::vector<std::vector<std::string>>& windows, BatchSize& batch_size)
+void generate_bonito_long_reads(std::vector<std::vector<std::string>>& windows, BatchSize& batch_size)
 {
-    constexpr uint32_t random_seed = 5827349;
-    std::minstd_rand rng(random_seed);
-
-    const int16_t number_of_windows = 2;
-    const int16_t number_of_reads   = 5;
-    const int32_t read_length       = 10000;
-    int32_t max_sequence_length     = read_length + 1;
-
-    std::vector<std::pair<int, int>> variation_ranges;
-    variation_ranges.push_back(std::pair<int, int>(30, 50));
-    variation_ranges.push_back(std::pair<int, int>(300, 500));
-    variation_ranges.push_back(std::pair<int, int>(1000, 1300));
-    variation_ranges.push_back(std::pair<int, int>(2000, 2200));
-    variation_ranges.push_back(std::pair<int, int>(3000, 3500));
-    variation_ranges.push_back(std::pair<int, int>(4000, 4200));
-    variation_ranges.push_back(std::pair<int, int>(5000, 5400));
-    variation_ranges.push_back(std::pair<int, int>(6000, 6200));
-    variation_ranges.push_back(std::pair<int, int>(8000, 8300));
-
-    std::vector<std::string> long_reads(number_of_reads);
-
-    for (int w = 0; w < number_of_windows; w++)
+    const std::string input_data = std::string(CUDAPOA_BENCHMARK_DATA_DIR) + "/sample-bonito.txt";
```

Comment:
since this logic is now almost the same for short reads as well, let's move it to a single function and instead just change the name of file used to generate the windows vector based on long/short read option.

```
void generate_window_data(filename, ...)
{
    parse_window_file(...)
    // for loop to calculate longest sequence, etc.
}

if (long_read)
   generate_window_data(bonito_file,...)
else
   generate_window_data(short_read_file,...)
```

Reply:
yes, the reason it is remained separate was initially they were quite different, but you're right, at this point, we can compact them into one function.

Reply:
done.